transportable-udfs-annotation-processor/src/test/resources/udfs/InnerClassUDF.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package udfs;
+
+import com.google.common.collect.ImmutableList;
+import com.linkedin.transport.api.data.StdString;
+import com.linkedin.transport.api.udf.StdUDF0;
+import com.linkedin.transport.api.udf.TopLevelStdUDF;
+import java.util.List;
+
+
+public class InnerClassUDF {
```

Comment:
Should use a different name since this is actually the outer :)

Reply:
Fixed